diff --git "a/C:\\Users\\wofs\\AppData\\Local\\Temp\\TortoiseGit\\multilog-5a5d0d3.000.pas" "b/C:\\lazarus\\components\\multilog\\multilog.pas"
index 072fb9f..793321a 100644
--- "a/C:\\Users\\wofs\\AppData\\Local\\Temp\\TortoiseGit\\multilog-5a5d0d3.000.pas"
+++ "b/C:\\lazarus\\components\\multilog\\multilog.pas"
@@ -70,14 +70,14 @@ const
 
   //LogClasses, convention with lc prefix
   //it's possible to define the constants to suit any need
-  lcAll = [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31];
-  lcDebug = 0;
-  lcError = 1;
-  lcInfo = 2;
-  lcWarning = 3;
-  lcEvents = 4;
-  //reserved
-  lcUser = 8;
+  //lcAll = [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31];
+  //lcDebug = 0;
+  //lcError = 1;
+  //lcInfo = 2;
+  //lcWarning = 3;
+  //lcEvents = 4;
+  ////reserved
+  //lcUser = 8;
   
 type
   TLogger = class;
